src/views/Profile.vue

```diff
@@ -266,21 +281,27 @@ a {
 }
 
 .personal-status {
-  table-layout: fixed;
-  margin-left: 2rem;
-  width: calc(300px - 2rem);
+  display: grid;
+  height: 5rem;
+  margin-left: 1.75rem;
+  grid-template-columns: calc(5rem + 18px) 64px 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
 
-  &:first-child > tr td:first-of-type {
-    box-sizing: content-box;
-    width: 4rem;
+  // align all texts to center
+  & * {
+    display: flex;
+    align-items: center;
   }
 
-  td {
-    padding: 0.625rem 1.5rem 0 0;
+  // avatar
+  :first-child {
+    grid-column: 1 / 2;
+    grid-row: 1/4;
   }
 
-  tr:first-of-type td {
-    padding-top: 0;
+  // Display Name
+  :nth-child(2) {
+    grid-column: 2 / 4;
   }
 }
```

Comment:
Maybe we can use bootstrap b-row b-col to handle flex box, instead of grid, which is not supported in legacy browsers, refers to: [https://caniuse.com/css-grid](https://caniuse.com/css-grid), [https://caniuse.com/?search=flex](https://caniuse.com/?search=flex)

Reply:
Resolved! Please check.